examples/devopness-api-client-vue/src/services/login.js

```diff
@@ -0,0 +1,15 @@
+import { DevopnessApiClient } from 'devopness-sdk-js'
+
+const DEVOPNESS_API_BASE_URL = 'https://test-api.devopness.com'
+const devopnessApi = new DevopnessApiClient({ baseUrl: DEVOPNESS_API_BASE_URL })
+
```

Comment:
As this file is named `login.js`, I'd suggest moving the SDK initialization to a common shared file.

Reply:
of course, I will start the change.